internal/certprovider.go

```diff
@@ -133,23 +133,26 @@ func (cp *NitroCertProvider) Roots() (*x509.CertPool, error) {
 	return cp.rootCerts, nil
 }
 
-// TODO: Remove SelfSignedCertProvider as part of
-//  https://blocky.atlassian.net/browse/BKY-5620
-
+// Deprecated: Remove selfSignedCertDER as part of https://blocky.atlassian.net/browse/BKY-5620
+//
```

Comment:
This extra line is added by go format in front of `go:embed`. We also need to put `go:embed` last in the comment stack